assets/base.css

```diff
@@ -1625,19 +1625,13 @@ input[type='checkbox'] {
 
 .announcement-bar__link .icon-arrow {
   display: inline-block;
-  height: 1.2rem;
+  width: 1.5rem;
   pointer-events: none;
-  padding-left: 1rem;
+  margin-left: 0.8rem;
   vertical-align: middle;
   margin-bottom: 0.2rem;
 }
```

Comment:
Updated! I also updated `mobile-facets__close-button` and  `.menu-drawer` to be 1.5rem since they are also icon arrows (previously 1.3rem). Should I tag the design team to take a look?

Reply:
Cuts out some lines of CSS too, which is 👍

Reply:
@Oliviammarcello @melissaperreault @wiktoriaswiecicka Do you mind taking a look :)

Reply:
The announcement bar arrow looks great! 🚢

Reply:
Looks great! 🚢